packages/slice-machine/server/src/api/index.ts

```
@@ -168,6 +171,36 @@ router.get(
   }
 );
 
+router.post(
+  "/auth/start",
+  async function (_req: express.Request, res: express.Response) {
+    const payload = await startAuth();
+    if (payload.err) {
+      return res.status(400).json({});
+    }
+    return res.status(200).json({});
+  }
+);
+
+router.post(
+  "/auth/status",
+  async function (_req: express.Request, res: express.Response) {
+    const payload = await statusAuth();
+    return res.status(200).json(payload);
```

I would return an error if the status was an error.

---

Yeah good one :)